repo/packages/K/kong/1/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "packagingVersion": "3.0",
+  "name": "kong",
+  "version": "0.10.3",
+  "scm": "https://github.com/Mashape/kong",
+  "maintainer": "https://getkong.org/community/",
```

Comment:
@coopr If we're only adding the open version of Kong, then yes it works. Lets discuss more about Mashape Enterprise offline.